Tests/XCUITests/DesktopModeTests.swift

```diff
@@ -9,7 +9,7 @@ import XCTest
 // Tests for both platforms
 class DesktopModeTestsIpad: IpadOnlyTestCase {
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2306852
-    func testLongPressReload() {
+    func testLongPressReloadMobile() {
```

Comment:
I'm not sure about this change... it is to request desktop view on iPad and the other test below is the same for iPhone.

I don't know either why it is running as part of the smoketest, it is not marked as so in TestRail:
https://testrail.stage.mozaws.net/index.php?/suites/view/45443&group_by=cases:section_id&group_order=asc&group_id=426801 

I'm fine with adding it to the smoketest by editing the test in testrail or removing it from the smoketest plan

Reply:
Undo the changes regarding the names and added the test in the smoke test list. Also updated test case in TR.